Description: Mention the -gui and -cli commands in the manpage for octave
Author: Rafael Laboissiere <rafael@laboissiere.net>
Last-Update: 2014-01-02

--- octave-3.8.0~rc2.orig/doc/interpreter/octave.1
+++ octave-3.8.0~rc2/doc/interpreter/octave.1
@@ -30,10 +30,13 @@
 octave \- A high-level interactive language for numerical computations.
 .SH SYNOPSIS
 \fBoctave\fP [\fIoptions\fP]... [\fIfile\fP]
+.br
+\fBoctave-cli\fP [\fIoptions\fP]... [\fIfile\fP]
 .SH DESCRIPTION
 Octave is a high-level language, primarily intended for numerical
-computations.  It provides a convenient command line interface for
-solving linear and nonlinear problems numerically.
+computations.  It provides convenient command line (\fBoctave-cli\fP command)
+and graphical user (\fBoctave\fP command) interfaces for solving linear and
+nonlinear problems numerically.
 .SH OPTIONS
 The complete set of command-line options for \fBoctave\fP is available by
 running the following command from the shell.
